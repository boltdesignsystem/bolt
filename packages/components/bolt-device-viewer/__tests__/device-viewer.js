import { render, stopServer } from '../../../testing/testing-helpers';
import schema from '../device-viewer.schema';
const { deviceName, color, orientation } = schema.properties;

describe('<bolt-device-viewer> Component', () => {
  afterAll(async () => {
    await stopServer();
  });

  test('basic usage', async () => {
    const results = await render(
      '@bolt-components-device-viewer/device-viewer.twig',
      {
        device: 'iphone8',
        orientation: 'portrait',
        color: 'white',
        image: {
          src: '/fixtures/device-screenshot--phone.jpg',
        },
      },
    );
    expect(results.ok).toBe(true);
    expect(results.html).toMatchSnapshot();
  });

  deviceName.enum.forEach(async deviceChoice => {
    test(`decvice-viewer display: ${deviceChoice}`, async () => {
      const results = await render(
        '@bolt-components-device-viewer/device-viewer.twig',
        {
          device: deviceChoice,
        },
      );
      expect(results.ok).toBe(true);
      expect(results.html).toMatchSnapshot();
    });
  });

  color.enum.forEach(async colorChoice => {
    test(`device-viewer display: ${colorChoice}`, async () => {
      const results = await render(
        '@bolt-components-device-viewer/device-viewer.twig',
        {
          device: 'iphone8',
          color: colorChoice,
        },
      );
      expect(results.ok).toBe(true);
      expect(results.html).toMatchSnapshot();
    });
  });

  orientation.enum.forEach(async orientationChoice => {
    test(`device-viewer display: ${orientationChoice}`, async () => {
      const results = await render(
        '@bolt-components-device-viewer/device-viewer.twig',
        {
          device: 'iphone8',
          orientation: orientationChoice,
        },
      );
      expect(results.ok).toBe(true);
      expect(results.html).toMatchSnapshot();
    });
  });

  test('usage with magnification', async () => {
    const results = await render(
      '@bolt-components-device-viewer/device-viewer.twig',
      {
        device: 'ipad',
        orientation: 'portrait',
        color: 'white',
        image: {
          src: '/fixtures/device-screenshot--tablet-portrait.jpg',
        },
        magnify: true,
      },
    );
    expect(results.ok).toBe(true);
    expect(results.html).toMatchSnapshot();
  });
});
