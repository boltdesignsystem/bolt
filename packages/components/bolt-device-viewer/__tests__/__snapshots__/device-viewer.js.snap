// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<bolt-device-viewer> Component basic usage 1`] = `
<bolt-device-viewer class="c-bolt-iphone8-viewer c-bolt-device-viewer marvel-device c-bolt-iphone8-viewer--portrait">
  <div class="c-bolt-device-viewer__inner c-bolt-iphone8-viewer__inner">
    <div class="c-bolt-iphone8-viewer__top-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__bottom-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__sensor">
    </div>
    <div class="c-bolt-iphone8-viewer__sleep">
    </div>
    <div class="c-bolt-iphone8-viewer__volume">
    </div>
    <div class="c-bolt-device-viewer__camera c-bolt-iphone8-viewer__camera">
    </div>
    <div class="c-bolt-iphone8-viewer__speaker">
    </div>
    <div class="c-bolt-device-viewer__home c-bolt-iphone8-viewer__home">
    </div>
    <div class="c-bolt-device-viewer__screen c-bolt-iphone8-viewer__screen">
      <bolt-image ratio="3000/5336"
                  placeholder-color="hsl(233, 33%, 97%)"
                  placeholder-image="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
                  style="background-color: hsl(233, 33%, 97%);"
                  src="/fixtures/device-screenshot--phone.jpg"
                  srcset="/fixtures/device-screenshot--phone-50.jpg 50w, /fixtures/device-screenshot--phone-100.jpg 100w, /fixtures/device-screenshot--phone-200.jpg 200w, /fixtures/device-screenshot--phone-320.jpg 320w, /fixtures/device-screenshot--phone-480.jpg 480w, /fixtures/device-screenshot--phone-640.jpg 640w, /fixtures/device-screenshot--phone-800.jpg 800w, /fixtures/device-screenshot--phone-1024.jpg 1024w, /fixtures/device-screenshot--phone-1366.jpg 1366w, /fixtures/device-screenshot--phone-1536.jpg 1536w, /fixtures/device-screenshot--phone-1920.jpg 1920w, /fixtures/device-screenshot--phone-2560.jpg 2560w, /fixtures/device-screenshot--phone-2880.jpg 2880w, /fixtures/device-screenshot--phone.jpg 3000w"
      >
        <bolt-ratio ratio="375/667">
          <ssr-keep for="bolt-ratio"
                    class="c-bolt-ratio"
                    style="padding-bottom: calc((667 / 375) * 100%); --aspect-ratio: 0.56221889055472;"
          >
            <img class="c-bolt-image__image-placeholder"
                 alt
                 src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
            >
            <img class="c-bolt-image__image c-bolt-image__lazyload c-bolt-image__lazyload--fade c-bolt-image__lazyload--blur js-lazyload"
                 alt
                 data-srcset="/fixtures/device-screenshot--phone-50.jpg 50w, /fixtures/device-screenshot--phone-100.jpg 100w, /fixtures/device-screenshot--phone-200.jpg 200w, /fixtures/device-screenshot--phone-320.jpg 320w, /fixtures/device-screenshot--phone-480.jpg 480w, /fixtures/device-screenshot--phone-640.jpg 640w, /fixtures/device-screenshot--phone-800.jpg 800w, /fixtures/device-screenshot--phone-1024.jpg 1024w, /fixtures/device-screenshot--phone-1366.jpg 1366w, /fixtures/device-screenshot--phone-1536.jpg 1536w, /fixtures/device-screenshot--phone-1920.jpg 1920w, /fixtures/device-screenshot--phone-2560.jpg 2560w, /fixtures/device-screenshot--phone-2880.jpg 2880w, /fixtures/device-screenshot--phone.jpg 3000w"
                 data-sizes="auto"
                 srcset="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
                 src="/fixtures/device-screenshot--phone.jpg"
            >
            <noscript>
              <img class="c-bolt-image__image"
                   alt
                   src="/fixtures/device-screenshot--phone.jpg"
                   srcset="/fixtures/device-screenshot--phone-50.jpg 50w, /fixtures/device-screenshot--phone-100.jpg 100w, /fixtures/device-screenshot--phone-200.jpg 200w, /fixtures/device-screenshot--phone-320.jpg 320w, /fixtures/device-screenshot--phone-480.jpg 480w, /fixtures/device-screenshot--phone-640.jpg 640w, /fixtures/device-screenshot--phone-800.jpg 800w, /fixtures/device-screenshot--phone-1024.jpg 1024w, /fixtures/device-screenshot--phone-1366.jpg 1366w, /fixtures/device-screenshot--phone-1536.jpg 1536w, /fixtures/device-screenshot--phone-1920.jpg 1920w, /fixtures/device-screenshot--phone-2560.jpg 2560w, /fixtures/device-screenshot--phone-2880.jpg 2880w, /fixtures/device-screenshot--phone.jpg 3000w"
              >
            </noscript>
          </ssr-keep>
        </bolt-ratio>
      </bolt-image>
    </div>
  </div>
</bolt-device-viewer>
`;

exports[`<bolt-device-viewer> Component decvice-viewer display: ipad 1`] = `
<bolt-device-viewer class="c-bolt-ipad-viewer c-bolt-device-viewer marvel-device c-bolt-ipad-viewer--black c-bolt-ipad-viewer--portrait">
  <div class="c-bolt-device-viewer__inner c-bolt-ipad-viewer__inner">
    <div class="c-bolt-ipad-viewer__top-bar">
    </div>
    <div class="c-bolt-ipad-viewer__bottom-bar">
    </div>
    <div class="c-bolt-ipad-viewer__sensor">
    </div>
    <div class="c-bolt-ipad-viewer__sleep">
    </div>
    <div class="c-bolt-ipad-viewer__volume">
    </div>
    <div class="c-bolt-device-viewer__camera c-bolt-ipad-viewer__camera">
    </div>
    <div class="c-bolt-ipad-viewer__speaker">
    </div>
    <div class="c-bolt-device-viewer__home c-bolt-ipad-viewer__home">
    </div>
    <div class="c-bolt-device-viewer__screen c-bolt-ipad-viewer__screen">
    </div>
  </div>
</bolt-device-viewer>
`;

exports[`<bolt-device-viewer> Component decvice-viewer display: iphone8 1`] = `
<bolt-device-viewer class="c-bolt-iphone8-viewer c-bolt-device-viewer marvel-device c-bolt-iphone8-viewer--black c-bolt-iphone8-viewer--portrait">
  <div class="c-bolt-device-viewer__inner c-bolt-iphone8-viewer__inner">
    <div class="c-bolt-iphone8-viewer__top-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__bottom-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__sensor">
    </div>
    <div class="c-bolt-iphone8-viewer__sleep">
    </div>
    <div class="c-bolt-iphone8-viewer__volume">
    </div>
    <div class="c-bolt-device-viewer__camera c-bolt-iphone8-viewer__camera">
    </div>
    <div class="c-bolt-iphone8-viewer__speaker">
    </div>
    <div class="c-bolt-device-viewer__home c-bolt-iphone8-viewer__home">
    </div>
    <div class="c-bolt-device-viewer__screen c-bolt-iphone8-viewer__screen">
    </div>
  </div>
</bolt-device-viewer>
`;

exports[`<bolt-device-viewer> Component decvice-viewer display: macbook 1`] = `
<bolt-device-viewer class="c-bolt-macbook-viewer c-bolt-device-viewer marvel-device c-bolt-macbook-viewer--black c-bolt-macbook-viewer--portrait">
  <div class="c-bolt-device-viewer__inner c-bolt-macbook-viewer__inner">
    <div class="c-bolt-macbook-viewer__top-bar">
    </div>
    <div class="c-bolt-macbook-viewer__bottom-bar">
    </div>
    <div class="c-bolt-macbook-viewer__sensor">
    </div>
    <div class="c-bolt-macbook-viewer__sleep">
    </div>
    <div class="c-bolt-macbook-viewer__volume">
    </div>
    <div class="c-bolt-device-viewer__camera c-bolt-macbook-viewer__camera">
    </div>
    <div class="c-bolt-macbook-viewer__speaker">
    </div>
    <div class="c-bolt-device-viewer__home c-bolt-macbook-viewer__home">
    </div>
    <div class="c-bolt-device-viewer__screen c-bolt-macbook-viewer__screen">
    </div>
  </div>
</bolt-device-viewer>
`;

exports[`<bolt-device-viewer> Component device-viewer display: black 1`] = `
<bolt-device-viewer class="c-bolt-iphone8-viewer c-bolt-device-viewer marvel-device c-bolt-iphone8-viewer--black c-bolt-iphone8-viewer--portrait">
  <div class="c-bolt-device-viewer__inner c-bolt-iphone8-viewer__inner">
    <div class="c-bolt-iphone8-viewer__top-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__bottom-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__sensor">
    </div>
    <div class="c-bolt-iphone8-viewer__sleep">
    </div>
    <div class="c-bolt-iphone8-viewer__volume">
    </div>
    <div class="c-bolt-device-viewer__camera c-bolt-iphone8-viewer__camera">
    </div>
    <div class="c-bolt-iphone8-viewer__speaker">
    </div>
    <div class="c-bolt-device-viewer__home c-bolt-iphone8-viewer__home">
    </div>
    <div class="c-bolt-device-viewer__screen c-bolt-iphone8-viewer__screen">
    </div>
  </div>
</bolt-device-viewer>
`;

exports[`<bolt-device-viewer> Component device-viewer display: gold 1`] = `
<bolt-device-viewer class="c-bolt-iphone8-viewer c-bolt-device-viewer marvel-device c-bolt-iphone8-viewer--gold c-bolt-iphone8-viewer--portrait">
  <div class="c-bolt-device-viewer__inner c-bolt-iphone8-viewer__inner">
    <div class="c-bolt-iphone8-viewer__top-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__bottom-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__sensor">
    </div>
    <div class="c-bolt-iphone8-viewer__sleep">
    </div>
    <div class="c-bolt-iphone8-viewer__volume">
    </div>
    <div class="c-bolt-device-viewer__camera c-bolt-iphone8-viewer__camera">
    </div>
    <div class="c-bolt-iphone8-viewer__speaker">
    </div>
    <div class="c-bolt-device-viewer__home c-bolt-iphone8-viewer__home">
    </div>
    <div class="c-bolt-device-viewer__screen c-bolt-iphone8-viewer__screen">
    </div>
  </div>
</bolt-device-viewer>
`;

exports[`<bolt-device-viewer> Component device-viewer display: landscape 1`] = `
<bolt-device-viewer class="c-bolt-iphone8-viewer c-bolt-device-viewer marvel-device c-bolt-iphone8-viewer--black c-bolt-iphone8-viewer--landscape">
  <div class="c-bolt-device-viewer__inner c-bolt-iphone8-viewer__inner">
    <div class="c-bolt-iphone8-viewer__top-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__bottom-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__sensor">
    </div>
    <div class="c-bolt-iphone8-viewer__sleep">
    </div>
    <div class="c-bolt-iphone8-viewer__volume">
    </div>
    <div class="c-bolt-device-viewer__camera c-bolt-iphone8-viewer__camera">
    </div>
    <div class="c-bolt-iphone8-viewer__speaker">
    </div>
    <div class="c-bolt-device-viewer__home c-bolt-iphone8-viewer__home">
    </div>
    <div class="c-bolt-device-viewer__screen c-bolt-iphone8-viewer__screen">
    </div>
  </div>
</bolt-device-viewer>
`;

exports[`<bolt-device-viewer> Component device-viewer display: portrait 1`] = `
<bolt-device-viewer class="c-bolt-iphone8-viewer c-bolt-device-viewer marvel-device c-bolt-iphone8-viewer--black c-bolt-iphone8-viewer--portrait">
  <div class="c-bolt-device-viewer__inner c-bolt-iphone8-viewer__inner">
    <div class="c-bolt-iphone8-viewer__top-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__bottom-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__sensor">
    </div>
    <div class="c-bolt-iphone8-viewer__sleep">
    </div>
    <div class="c-bolt-iphone8-viewer__volume">
    </div>
    <div class="c-bolt-device-viewer__camera c-bolt-iphone8-viewer__camera">
    </div>
    <div class="c-bolt-iphone8-viewer__speaker">
    </div>
    <div class="c-bolt-device-viewer__home c-bolt-iphone8-viewer__home">
    </div>
    <div class="c-bolt-device-viewer__screen c-bolt-iphone8-viewer__screen">
    </div>
  </div>
</bolt-device-viewer>
`;

exports[`<bolt-device-viewer> Component device-viewer display: silver 1`] = `
<bolt-device-viewer class="c-bolt-iphone8-viewer c-bolt-device-viewer marvel-device c-bolt-iphone8-viewer--silver c-bolt-iphone8-viewer--portrait">
  <div class="c-bolt-device-viewer__inner c-bolt-iphone8-viewer__inner">
    <div class="c-bolt-iphone8-viewer__top-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__bottom-bar">
    </div>
    <div class="c-bolt-iphone8-viewer__sensor">
    </div>
    <div class="c-bolt-iphone8-viewer__sleep">
    </div>
    <div class="c-bolt-iphone8-viewer__volume">
    </div>
    <div class="c-bolt-device-viewer__camera c-bolt-iphone8-viewer__camera">
    </div>
    <div class="c-bolt-iphone8-viewer__speaker">
    </div>
    <div class="c-bolt-device-viewer__home c-bolt-iphone8-viewer__home">
    </div>
    <div class="c-bolt-device-viewer__screen c-bolt-iphone8-viewer__screen">
    </div>
  </div>
</bolt-device-viewer>
`;

exports[`<bolt-device-viewer> Component usage with magnification 1`] = `
<bolt-device-viewer class="c-bolt-ipad-viewer c-bolt-device-viewer marvel-device c-bolt-ipad-viewer--portrait">
  <div class="c-bolt-device-viewer__inner c-bolt-ipad-viewer__inner">
    <div class="c-bolt-ipad-viewer__top-bar">
    </div>
    <div class="c-bolt-ipad-viewer__bottom-bar">
    </div>
    <div class="c-bolt-ipad-viewer__sensor">
    </div>
    <div class="c-bolt-ipad-viewer__sleep">
    </div>
    <div class="c-bolt-ipad-viewer__volume">
    </div>
    <div class="c-bolt-device-viewer__camera c-bolt-ipad-viewer__camera">
    </div>
    <div class="c-bolt-ipad-viewer__speaker">
    </div>
    <div class="c-bolt-device-viewer__home c-bolt-ipad-viewer__home">
    </div>
    <div class="c-bolt-device-viewer__screen c-bolt-ipad-viewer__screen">
      <bolt-image-zoom>
        <span class="c-bolt-image-zoom__tilt c-bolt-image-zoom__tilt--top-left">
        </span>
        <span class="c-bolt-image-zoom__tilt c-bolt-image-zoom__tilt--top-center">
        </span>
        <span class="c-bolt-image-zoom__tilt c-bolt-image-zoom__tilt--top-right">
        </span>
        <span class="c-bolt-image-zoom__tilt c-bolt-image-zoom__tilt--middle-left">
        </span>
        <span class="c-bolt-image-zoom__tilt c-bolt-image-zoom__tilt--middle-center">
        </span>
        <span class="c-bolt-image-zoom__tilt c-bolt-image-zoom__tilt--middle-right">
        </span>
        <span class="c-bolt-image-zoom__tilt c-bolt-image-zoom__tilt--bottom-left">
        </span>
        <span class="c-bolt-image-zoom__tilt c-bolt-image-zoom__tilt--bottom-center">
        </span>
        <span class="c-bolt-image-zoom__tilt c-bolt-image-zoom__tilt--bottom-right">
        </span>
        <bolt-image zoom
                    ratio="2048/2732"
                    placeholder-color="hsl(233, 33%, 97%)"
                    placeholder-image="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
                    style="background-color: hsl(233, 33%, 97%);"
                    src="/fixtures/device-screenshot--tablet-portrait.jpg"
                    srcset="/fixtures/device-screenshot--tablet-portrait-50.jpg 50w, /fixtures/device-screenshot--tablet-portrait-100.jpg 100w, /fixtures/device-screenshot--tablet-portrait-200.jpg 200w, /fixtures/device-screenshot--tablet-portrait-320.jpg 320w, /fixtures/device-screenshot--tablet-portrait-480.jpg 480w, /fixtures/device-screenshot--tablet-portrait-640.jpg 640w, /fixtures/device-screenshot--tablet-portrait-800.jpg 800w, /fixtures/device-screenshot--tablet-portrait-1024.jpg 1024w, /fixtures/device-screenshot--tablet-portrait-1366.jpg 1366w, /fixtures/device-screenshot--tablet-portrait-1536.jpg 1536w, /fixtures/device-screenshot--tablet-portrait-1920.jpg 1920w, /fixtures/device-screenshot--tablet-portrait.jpg 2048w"
        >
          <bolt-ratio ratio="512/683">
            <ssr-keep for="bolt-ratio"
                      class="c-bolt-ratio"
                      style="padding-bottom: calc((683 / 512) * 100%); --aspect-ratio: 0.74963396778917;"
            >
              <img class="c-bolt-image__image-placeholder"
                   alt
                   data-zoom="/fixtures/device-screenshot--tablet-portrait.jpg"
                   src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
              >
              <img class="c-bolt-image__image c-bolt-image__lazyload c-bolt-image__lazyload--fade c-bolt-image__lazyload--blur js-lazyload"
                   alt
                   data-srcset="/fixtures/device-screenshot--tablet-portrait-50.jpg 50w, /fixtures/device-screenshot--tablet-portrait-100.jpg 100w, /fixtures/device-screenshot--tablet-portrait-200.jpg 200w, /fixtures/device-screenshot--tablet-portrait-320.jpg 320w, /fixtures/device-screenshot--tablet-portrait-480.jpg 480w, /fixtures/device-screenshot--tablet-portrait-640.jpg 640w, /fixtures/device-screenshot--tablet-portrait-800.jpg 800w, /fixtures/device-screenshot--tablet-portrait-1024.jpg 1024w, /fixtures/device-screenshot--tablet-portrait-1366.jpg 1366w, /fixtures/device-screenshot--tablet-portrait-1536.jpg 1536w, /fixtures/device-screenshot--tablet-portrait-1920.jpg 1920w, /fixtures/device-screenshot--tablet-portrait.jpg 2048w"
                   data-sizes="auto"
                   srcset="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
                   src="/fixtures/device-screenshot--tablet-portrait.jpg"
                   data-zoom="/fixtures/device-screenshot--tablet-portrait.jpg"
              >
              <noscript>
                <img class="c-bolt-image__image"
                     alt
                     src="/fixtures/device-screenshot--tablet-portrait.jpg"
                     srcset="/fixtures/device-screenshot--tablet-portrait-50.jpg 50w, /fixtures/device-screenshot--tablet-portrait-100.jpg 100w, /fixtures/device-screenshot--tablet-portrait-200.jpg 200w, /fixtures/device-screenshot--tablet-portrait-320.jpg 320w, /fixtures/device-screenshot--tablet-portrait-480.jpg 480w, /fixtures/device-screenshot--tablet-portrait-640.jpg 640w, /fixtures/device-screenshot--tablet-portrait-800.jpg 800w, /fixtures/device-screenshot--tablet-portrait-1024.jpg 1024w, /fixtures/device-screenshot--tablet-portrait-1366.jpg 1366w, /fixtures/device-screenshot--tablet-portrait-1536.jpg 1536w, /fixtures/device-screenshot--tablet-portrait-1920.jpg 1920w, /fixtures/device-screenshot--tablet-portrait.jpg 2048w"
                >
              </noscript>
            </ssr-keep>
          </bolt-ratio>
        </bolt-image>
        <div class="c-bolt-image-zoom__overlay">
          <bolt-icon class="c-bolt-image-zoom__overlay-icon"
                     name="search"
                     size="xlarge"
                     background="circle"
                     aria-hidden="true"
          >
            <span class="c-bolt-icon c-bolt-icon--search c-bolt-icon--xlarge  has-background has-circle-background">
              <span class="c-bolt-icon__background-shape c-bolt-icon__background-shape--circle">
              </span>
              <svg class="c-bolt-icon__icon c-bolt-icon__icon--xlarge"
                   xmlns="http://www.w3.org/2000/svg"
                   viewbox="0 0 24 24"
              >
                <g fill="none"
                   fill-rule="evenodd"
                >
                  <path>
                  </path>
                  <path fill="var(--bolt-theme-icon, currentColor)"
                        d="M9.5 15a5.5 5.5 0 110-11 5.5 5.5 0 010 11m12.2 5.3L15.5 14c1-1.3 1.5-2.9 1.5-4.6a7.5 7.5 0 10-3 6l6.3 6.2a1 1 0 001.4-1.4"
                  >
                  </path>
                </g>
              </svg>
            </span>
          </bolt-icon>
        </div>
      </bolt-image-zoom>
    </div>
  </div>
</bolt-device-viewer>
`;
